modpack_format/implementing.md

```diff
@@ -0,0 +1,31 @@
+# Implementing the Modpack Format
+This is a guide for launchers on how to implement the modpack format.
+
+## Format Meta
+Format Meta, like `formatType` and `formatVersion` are meta-fields used for describing what format the file is in.
+
+### `formatType`
+If you encounter an unknown `formatType`, an error should be thrown and shown to the user. Installation of a modpack cannot continue of the `formatType` is unknown.
+
+### `formatVersion`
+You may want to add a special indicator if future versions are encountered in `formatVersion` - for example, the `formatVersion` field of a modpack is set to `2`, while the launcher only supports version `1`. As this is an unknown format to the launcher, you may want to make it clear with a message that it's an unsupported future version.
+
+## Files
+Loop through each file in the `files` array. You may want to do this in batches (e.g. 5 files at a time, concurrently) for efficiency.
+
+### `path`
+Each path of the file is relative to the Minecraft instance directory. For example `mods/MyMod.jar` resolves to `.minecraft/mods/MyMod.jar`. When parsing the `path` field, make sure it doesn't exit the Minecraft instance directory for security reasons.
```

Comment:
i.e. it should not contain a `..` component

Reply:
Added, thanks.